seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-kafka/src/main/java/org/apache/seatunnel/flink/kafka/sink/KafkaSink.java

```diff
@@ -67,12 +67,12 @@ public Config getConfig() {
 
     @Override
     public CheckResult checkConfig() {
-        return CheckConfigUtil.checkAllExists(config, "topics");
+        return CheckConfigUtil.checkAllExists(config, "topic");
     }
 
     @Override
     public void prepare(FlinkEnvironment env) {
-        topic = config.getString("topics");
```

Comment:
Considering the issue of compatibility, it is best to discuss it first

Reply:
For Flink engine, we should used `topics`

Reply:
OK